src/app/admin/websiteDataTable/fetchWebsiteData.ts

```diff
@@ -1,46 +1,79 @@
 import supabase from "@/supabaseClient";
-import { DatabaseError } from "@/app/errorClasses";
 import { WebsiteDataRow } from "./WebsiteDataTable";
 import { Tables } from "@/databaseTypesFile";
 import { logErrorReturnLogId } from "@/logger/logger";
+import { AuditLog, sendAuditLog } from "@/server/auditLog";
 
 type DbWebsiteData = Tables<"website_data">;
+type FetchWebsiteDataErrors = "Failed to fetch website data";
```

Comment:
to make it clear that this is not meant to be a user-facing message
```suggestion
type FetchWebsiteDataErrors = "failedToFetchWebsiteData";
```